store/src/java/com/zimbra/cs/index/event/logger/EventLogger.java

```diff
@@ -0,0 +1,25 @@
+package com.zimbra.cs.index.event.logger;
+
+import com.zimbra.common.service.ServiceException;
+import com.zimbra.common.util.ZimbraLog;
+import com.zimbra.cs.index.event.Event;
+
+public abstract class EventLogger {
+    public abstract void log(Event event);
+
+    protected static EventLogger eventLogger;
+
+    public static final void setEventLogger(Class<? extends EventLogger> loggerClass) throws ServiceException {
+        String className = loggerClass.getName();
+        ZimbraLog.store.info("setting EventLogger class %s", className);
```

Comment:
I think it would be worthwhile to make a new "event" logger category under ZimbraLog. That way we have the option of redirecting event logs to a different file, and it makes it easier to analyze logs during development.